run_runner_locally.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -e
+
+export PYTHON_BINARY=python3.9
+export RED='\033[0;31m'
+export NC='\033[0m' # No Color
+
+
+# Check if Python 3.9 or higher is installed
+if ! $PYTHON_BINARY -c "import sys; exit(not (sys.version_info.major == 3 and sys.version_info.minor >= 9))"; then
+  echo -e "${RED}Error: Python 3.9 or higher is not installed or was not run\n${NC}"
+  echo "You are using `$PYTHON_BINARY --version` located at `which $PYTHON_BINARY`"
+  echo "To change your Python version, edit the PYTHON_BINARY variable at the top of this script"
+  exit 1
+fi
+
+#if ! $PYTHON_BINARY -c "import poetry" &> /dev/null; then
+#  echo -e "Error: Poetry is not installed for the Python binary you chose.\n"
+#  echo "You are using `$PYTHON_BINARY --version` located at `which $PYTHON_BINARY`"
+#  echo "To change your Python version, edit the PYTHON_BINARY variable at the top of this script"
+#  exit 1
+#fi
+
+# Check if Poetry is installed globally
+if ! command -v poetry &> /dev/null
+then
+    echo -e "${RED}Poetry is not installed globally. Make sure the `poetry` command works${NC}"
+    exit 1
+fi
+
+echo "Python 3.9+ and Poetry are installed."
+echo "Now setting up Robusta"
+
+poetry env use $PYTHON_BINARY
+poetry install --extras=all
+echo "All dependencies installed"
+
+
+echo "Setting up local runner environment"
+mkdir -p deployment/playbooks/defaults
+ln -fsw $(pwd)/playbooks/robusta_playbooks/ ./deployment/playbooks/defaults
+ln -fsw $(pwd)/playbooks/pyproject.toml ./deployment/playbooks/defaults
+
+echo "Checking if runner can listen on port 5006"
```

Comment:
I would use port 5000.
I think the use of 5006 was because port 5000 was taken on a developer machine.
Typically it should be available.

Reply:
Fixed